cleanlab/datalab/internal/issue_manager/regression/label.py

```diff
@@ -58,50 +64,53 @@ def __init__(
         self,
         datalab: Datalab,
         clean_learning_kwargs: Optional[Dict[str, Any]] = None,
+        threshold: float = 0.1,
         health_summary_parameters: Optional[Dict[str, Any]] = None,
         **_,
     ):
         super().__init__(datalab)
         self.cl = CleanLearning(**(clean_learning_kwargs or {}))
-
-    @staticmethod
-    def _process_find_label_issues_kwargs(kwargs: Dict[str, Any]) -> Dict[str, Any]:
-        """Searches for keyword arguments that are meant for the
-        CleanLearning.find_label_issues method call
-
-        Examples
-        --------
-        >>> from cleanlab.datalab.internal.issue_manager.regression.label import LabelIssueManager
-        >>> RegressionLabelIssueManager._process_find_label_issues_kwargs({'coarse_search_range': [0.1, 0.9]})
-        {'coarse_search_range': [0.1, 0.9]}
-        """
-        accepted_kwargs = [
-            "uncertainty",
-            "coarse_search_range",
-            "fine_search_size",
-            "save_space",
-            "model_kwargs",
-        ]
-        return {k: v for k, v in kwargs.items() if k in accepted_kwargs and v is not None}
+        # This is a field for prioritizing features only when using a custom model
+        self._uses_custom_model = "model" in (clean_learning_kwargs or {})
+        self.threshold = threshold
 
     def find_issues(
         self,
-        features: np.ndarray,
+        features: Optional[np.ndarray] = None,
+        predictions: Optional[np.ndarray] = None,
         **kwargs,
     ) -> None:
         """Find label issues in the datalab."""
```

Comment:
you should write in english the priority order that happens inside here, eg:

If model is provided and features, then will use CleanLearning. Otherwise if pred_probs are provided, then will use them, etc etc. 

Otherwise this knowledge will be lost to time, if not explicitly documented

Reply:
Added a note

<img width="801" alt="image" src="https://github.com/cleanlab/cleanlab/assets/18127060/78fabf7a-a924-46c0-81d8-9b8afbc347db">